app/Http/Controllers/Api/UsersController.php

```diff
@@ -69,6 +69,10 @@ public function store(UserRequest $request)
             unset($credentials['term']);
 
             $user = $this->registrar->createUser((object) $credentials);
+
+            if ($request->input('term')) {
```

Comment:
I would add && `term = email` because I think you can pass multiple types of things in to get a user here